sunkit_pyvista/plotter.py

```diff
@@ -140,3 +144,38 @@ def plot_solar_axis(self, length=2.5, arrow_kwargs={}, **kwargs):
                          scale='auto',
                          **defaults)
         self.plotter.add_mesh(arrow, **kwargs)
+
+    def plot_field_lines(self, gong_map, lon, lat, radius, nrho=25, rss=2.5, **kwargs):
+        """
+        Plots the field lines from `pfsspy`.
+
+        Parameters
+        ----------
+        gong_map : `pfsspy.map.GongSynopticMap`
+            to produce the field lines
+        lon : `astropy.units.quantity`
+        lat : `astropy.units.quantity`
+        radius : `float`
+        nrho : `int`
+        rss : `int`
+        **kwargs :
+            Keyword arguments are handed to `pyvista.Plotter.add_mesh`.
+        """
+        tracer = tracing.PythonTracer()
+        input_ = pfsspy.Input(gong_map, nrho, rss)
+        output_ = pfsspy.pfss(input_)
+
+        seeds = SkyCoord(lon, lat, radius*R_sun,
+                         frame=gong_map.coordinate_frame)
+
+        field_lines = tracer.trace(seeds, output_)
+        grid_data = []
+
+        for field_line in field_lines:
+            grid = self._coords_to_xyz(field_line.coords.ravel())
```

Comment:
Hmm, do you need to do `.ravel()` here? It should work without, but if it doesn't you have probably found a pfsspy bug.